src/main/java/school/lemon/changerequest/java/io/pr2/JsonObject.java

```diff
@@ -0,0 +1,31 @@
+package school.lemon.changerequest.java.io.pr2;
+
+import java.util.Map;
+
+/**
+ * Created by lera on 10.02.17.
+ */
+public class JsonObject implements JsonType {
+    private final Map<String, JsonType> map;
+
+    public JsonObject(Map<String, JsonType> map) {
+        this.map = map;
+    }
+
+    public Map<String, JsonType> getMap() {
+        return map;
+    }
+
+    @Override
+    public String toStringRepresentation() {
+        if (map.isEmpty()) return "{}";
+        StringBuilder str = new StringBuilder();
+        str.append("{");
+        for (Map.Entry<String, JsonType> pair : map.entrySet()) {
+            str.append("\"").append(pair.getKey()).append("\"").append(":").append(" ").append(pair.getValue().toStringRepresentation()).append(",");
```

Comment:
Both `key` and `value` of `pair` could be `null`.
Please add additional NPE checks.